cg/meta/upload/fohm/fohm.py

```diff
@@ -268,3 +281,12 @@ def update_uploaded_at(self, case_id: str) -> None:
         case_obj: models.Family = self.status_db.family(case_id)
         case_obj.analyses[0].uploaded_at = dt.datetime.now()
         self.status_db.commit()
+
+    def delete_empty_directories(self) -> None:
```

Comment:
I'm not sure how i feel about the upload command pruning the fohm folder of all empty directories. Does it not make more sense to have each upload clean its own directory?

Reply:
That is true, it does! I guess you could just delete the entire ```daily_rawdata_path``` directory and ```daily_report_path``` when all of the files in them have been delivered. But it makes me a bit uneasy, seems safer to delete each file individually after delivering it and then attempt to delete the entire directory.

Reply:
I think both approaches are valid. And if you prefer removing empty directories I'm fine with it 👍 Where are we planning to remove `daily_bundle_path`?

Reply:
Nice catch